configure.ac

```diff
@@ -28,6 +29,19 @@ if ! test -r "$srcdir"/pire/inline.cpp && test x"$LEX" = "x:"; then
 	AC_MSG_ERROR([[flex/lex not found (and no prebuilt pire-inline lexer used)]])
 fi
 
+# python binding setup
+AC_DISABLE_STATIC
```

Comment:
It changes the default behavior of libtool; it appears that this macro is not necessary for python extensions -- I'll remove it